modules/services/flameshot.nix

```diff
@@ -33,6 +33,7 @@ in
       };
 
       Service = {
+        Environment = "PATH=${package}/bin:${pkgs.qt510.qtbase}/bin";
```

Comment:
Maybe it would be more future proof to do as in https://github.com/rycee/home-manager/pull/259, i.e., to have

```
Environment = "PATH=%h/.nix-profile/bin";
```

here since then it should work even if flameshot starts needing new qt plugins. See https://github.com/rycee/home-manager/pull/259 for @uvNikita's fix of similar services.

Reply:
Wouldn't that open the door for even more impurities (like this issue was apparently only noticed when trying the service on a non-kde-machine)?

I mean, if you're fine with it I'm fine with it, too, but are you sure you want to go down that road?

Reply:
Yeah, it's not ideal but it is my understanding that this is the preferred way to handle Qt applications. See https://github.com/rycee/home-manager/issues/249#issuecomment-388637480.

Reply:
That said, if one is certain that all plugins are accounted for then having them in the `PATH` seems like a nice solution to me. Perhaps could have a function that collects the bin paths of the propagated build inputs for the package instead of doing it manually?

Reply:
Ah, I think I remember Qt doing some form of sharing of the plugins across processes and if you have multiple copies of the plugins loaded then bad things happen. So I guess that might be a reason why it is best to install all Qt programs into the profile so that they are guaranteed to share the same plugins?